data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -1,12 +1,24 @@
-'''
-The Reverse Polish Nation also known as Polish postfix notation or simply postfix notation
-https://en.wikipedia.org/wiki/Reverse_Polish_notation
-Classic examples of simple stack implementations
-Valid operators are +, -, *, /. Each operand may be an integer or another expression.
-'''
+# The Reverse Polish Nation also known as Polish postfix notation or simply postfix notation
+# https://en.wikipedia.org/wiki/Reverse_Polish_notation
+# Classic examples of simple stack implementations
+# Valid operators are +, -, *, /. Each operand may be an integer or another expression.
 
 
 def evaluate_postfix(postfix_notation):
+    """
+    >>> array = ["10", "6", "9", "3", "+", "-11", "*", "/", "*", "17", "+", "5", "+"]
+    >>> evaluate_postfix(array)
+    22
+    >>> array = ["2", "1", "+", "3", "*"]
+    >>> evaluate_postfix(array)
```

Comment:
```suggestion
    >>> evaluate_postfix(["2", "1", "+", "3", "*"])
```
Variables that go out of scope on the very next line should only be used if:
1. they shorten very long lines of code, or
2. their variable name helps to clarify something that is not obvious.

`array` does neither.

Repeat below...